### ISSUE TITLE
Could not find file with URI because it is a relative path

### ISSUE THREAD

**Issue by sunyuyangg555** [ManyDesigns/Portofino#582]
Hi, this is a awesome project,  so I want to try it. I download the zip file from https://sourceforge.net/projects/portofino/
then I click the startup.bat to start, but I encounter a error:
```
2022-10-09 15:53:26,279 ERROR c.m.p.dispatcher.AbstractResource          Could not access child: :auth
org.apache.commons.vfs2.FileSystemException: Could not find file with URI "/Users/sunjian/portofino-5.3.2/apache-tomcat-9.0.62/webapps/ROOT/WEB-INF/actions/login" because it is a relative path, and no base URI was provided.
        at org.apache.commons.vfs2.FileSystemException.requireNonNull(FileSystemException.java:88)
        at org.apache.commons.vfs2.impl.DefaultFileSystemManager.resolveFile(DefaultFileSystemManager.java:743)
        at org.apache.commons.vfs2.impl.DefaultFileSystemManager.resolveFile(DefaultFileSystemManager.java:692)
        at org.apache.commons.vfs2.impl.DefaultFileSystemManager.resolveFile(DefaultFileSystemManager.java:648)
        at com.manydesigns.portofino.resourceactions.AbstractResourceAction.getChildLocation(AbstractResourceAction.java:179)
        at com.manydesigns.portofino.dispatcher.AbstractResource.consumePathSegment(AbstractResource.java:98)
        at com.manydesigns.portofino.dispatcher.AbstractResourceWithParameters.consumePathSegment(AbstractResourceWithParameters.java:54)
        at com.manydesigns.portofino.rest.PortofinoRoot.consumePathSegment(PortofinoRoot.java:94)
        at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.lang.reflect.Method.invoke(Method.java:498)
        at org.glassfish.jersey.server.internal.routing.SubResourceLocatorRouter.lambda$getResource$0(SubResourceLocatorRouter.java:132)
```

- OS: winodws 10
- version: 5.3.2

What should I do? thanks very much!

**Comment by alessiostalla** [comment]
Hi, thanks for trying Portofino out!
It's a known issue on Windows: #566 
We're working on releasing a new version to fix this. Sorry for the inconvenience!

**Comment by sunyuyangg555** [comment]
thank you for your quick reply!

**Comment by alessiostalla** [comment]
Hi @sunyuyangg555 , we've released Portofino 5.3.3, it should fix the issue. Let me know if it persists or if you find other issues!

**Comment by sunyuyangg555** [comment]
I'm sorry for replying so late. I have downloaded the 5.3.3. Now all works. I will continue to try, thanks very much!
